pages/web-literacy/WebLiteracyPage.jsx

```diff
@@ -1,44 +1,314 @@
 var React = require('react');
 var OutboundLink = require('react-ga').OutboundLink;
+var ReactRouter = require('react-router');
+var History = ReactRouter.History;
+var Link = ReactRouter.Link;
 
 var Illustration = require('../../components/illustration.jsx');
 
 var webmaps = require('./webmaplisting.jsx');
 
-var WebLiteracyPage = React.createClass({
+var CircleTree = require('react-circletree/es5');
+var weblitdataroot = require('./weblitdata');
+var weblitdata = weblitdataroot["WEB LITERACY"];
+var weblitcontent = require('./weblitcontent');
+var activitydata = require('./activitydata');
+var categories = require('./categories');
+var weblitcolors = require('./colors');
+var topicContent = weblitcontent.topics;
+var webLitSkillsContent = weblitcontent.webLitSkills;
+var makeLinksFromWebLitSkills = require("./MakeLinksFromWebLitSkills.jsx");
+
+function makeLinksFrom21CSkills(skills21C) {
+  return skills21C.map(function(skill21C, index) {
+    return (
+      <span key={skill21C} className="comma-separated-links">
```

Comment:
do we need the span? It doesn't really do anything to the content (it just wraps inline content as inline content) and we can set those classes directly on the `<a>`